### PR TITLE
Power VS: Add zones

### PR DESCRIPTION
**Pull request by mjturek**

### CODE REVIEW

File: pkg/tfvars/powervs/powervs.go [hunk 1]
```diff
@@ -3,34 +3,22 @@ package powervs
...
-}
-
 type config struct {
 	ServiceInstanceID    string `json:"powervs_cloud_instance_id"`
 	APIKey               string `json:"powervs_api_key"`
 	SSHKey               string `json:"powervs_ssh_key"`
 	PowerVSRegion        string `json:"powervs_region"`
 	PowerVSZone          string `json:"powervs_zone"`
 	VPCRegion            string `json:"powervs_vpc_region"`
+	VPCZone              string `json:"powervs_vpx_zone"`
```
Comment (clnperez) [inline pkg/tfvars/powervs/powervs.go:21]: there's a typo here (x instead of c)

File: pkg/rhcos/powervs_regions.go [hunk 2]
```diff
@@ -2,69 +2,74 @@ package rhcos
...
-		"lon06",
-	},
-	"mon":     []string{"mon01"},
-	"osa":     []string{"osa21"},
-	"syd":     []string{"syd04"},
-	"sao":     []string{"sao01"},
-	"tor":     []string{"tor01"},
-	"tok":     []string{"tok04"},
-	"us-east": []string{"us-east"},
+// PowerVSRegion describes resources associated with a region in powervs.
```
Comment (clnperez) [inline pkg/rhcos/powervs_regions.go:5]: This comment should end with "Power VS" instead of "powervs"

File: pkg/rhcos/powervs_regions.go [hunk 3]
```diff
@@ -2,69 +2,74 @@ package rhcos
 
 // TODO(cklokman): This entire file should be programatically generated similar to aws
```
Comment (clnperez) [inline pkg/rhcos/powervs_regions.go:3]: can you delete this while you're editing this file?

**Review by clnperez — CHANGES_REQUESTED** [review]
a couple more requests i missed before

**Comment by mjturek** [comment]
This PR was pulled into #5224 with the changes requested by @clnperez . Closing.

**Pull request closed without merging**